lint/linter.py

```diff
@@ -1111,7 +1127,9 @@ def clear_view(view):
 
         view.erase_status(STATUS_KEY)
         highlight.clear_view(view)
-        persist.errors.pop(view.id(), None)
+        vid = view.buffer_id()
+        if vid in persist.raw_errors:
+            del persist.raw_errors[vid]
 
```

Comment:
should be bid

Reply:
Actually this whole function should go away. Just popping bid doesn't work, bc you can have multiple views into the same bid. You can delete this only if it is the last view.

But otoh we just don't need this here.